compiler/rustc_session/src/options.rs

```diff
@@ -1488,6 +1485,9 @@ options! {
         "in diagnostics, use heuristics to shorten paths referring to items"),
     ui_testing: bool = (false, parse_bool, [UNTRACKED],
         "emit compiler diagnostics in a form suitable for UI testing (default: no)"),
+    uninit_const_chunk_threshold: usize = (256, parse_number, [TRACKED],
```

Comment:
This seems like a fairly high value, can we start something more conservative? Like 16 maybe?

Reply:
Sure. I do think we can get away with 256--I think problematic cases will be arrays with far more than 256 elements--but I can't think of a realistic case that would be improved by having the limit that high, so 16 is fine. It's easy to increase it later if we find something.